collector/tracking.go

```diff
@@ -91,6 +91,26 @@ var (
 		prometheus.GaugeValue,
 	}
 
+	trackingRootDelay = typedDesc{
+		prometheus.NewDesc(
+			prometheus.BuildFQName(namespace, trackingSubsystem, "root_delay_seconds"),
+			"Chrony tracking total of all network path delays to the NTP root",
```

Comment:
This one is short enough we can just copy-n-paste the man page help string.

```suggestion
			"This is the total of the network path delays to the stratum-1 computer from which the computer is ultimately synchronised",
```